### ISSUE TITLE
Get thumbnail from upload response?

### ISSUE THREAD

**Issue by HTMLGuyLLC** [pqina/filepond-plugin-image-preview#72]
I'm converting heic to jpg and want to update the frontend to display the preview but can't seem to figure out how. Do you have a workaround for this, or can you add an enhancement for later to do it?

**Comment by rikschennink** [comment]
I don't think this is currently possible, the source file is used for the preview, so if it's a HEIC it can't be used.

**Comment by HTMLGuyLLC** [comment]
I accomplished it for now by adding a background image when a canvas doesn't exist. Works fine
